docs/migrating_to_17.md

```diff
@@ -24,24 +26,49 @@ Key points:
 ```
 
 
-## 3. *(Optional)* Set up Grafana
+## 3. Set up credentials for aggregation/Beacon and, optionally, set up Grafana
 
-TODO: environment
+If you wish to enable Grafana, you first must enable the monitoring feature in your `local.env` file:
+
+```bash
+BENTO_MONITORING_ENABLED='true'
+```
+
+To create the client secrets for aggregation/Beacon and Grafana (if the latter is enabled), run the following commands:
 
 ```bash
 ./bentoctl.bash start auth
 ./bentoctl.bash init-auth
 ```
 
+Aggregation/Beacon data access authorization will not work until an authorization service grant is configured; 
```

Comment:
Remind to add the client secrets to the configuration file

Reply:
added